### ISSUE TITLE
Is your IP traceable when executed?

### ISSUE THREAD

**Issue by urmom2594** [MyLifeIsSakura/HelloFriendBash-Worm-HFB#2]
I saw that the worm uses the IP of the victim/host. So can you be traced via IP or is that something that I just happen to see while that isn't true?

**Comment by MyLifeIsSakura** [comment]
nope, that code finds ips on network and then copies itself onto ips desktop and shared files, P.S: that part of the code doesnt work anymore since now you need pesky admin privileges for that, https://github.com/MyLifeIsSakura/Batch-LAN-Worm has a code that does the same that works, so for this code to work all ur pc needs admin privilegles over all pcs in the network, like the pcs of IT guys in a school, otherwise wont work and will crash the script, i should cut that part of the code out.

**Comment by urmom2594** [comment]
thanks for your explanation, and fast response.

**Comment by MyLifeIsSakura** [comment]
i was just looking thru my github so thats the fast response:)
